### PR TITLE
Map including popovers with losses by state, GeoJSON API endpoints for sat data

### PR DESCRIPTION
**Pull request by skonkiel**
The popovers on the map now include estimated animal deaths, human deaths, and homes lost for each state.

I've also added GeoJSON API endpoints for the VIIRS and MODIS data. The formatting for the features seems to check out (I validated using http://geojsonlint.com/)

### CODE REVIEW

File: Resources/states.geojson [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
+  "type": "FeatureCollection",
```
Comment (philipstubbs13) [inline Resources/states.geojson:2]: Consider using git lfs to track this file type (geojson) in the future. The more we can leverage git lfs for data files, the more we gain in terms of speed in pushing and pulling changes from GitHub.

Comment (philipstubbs13) [reply Resources/states.geojson:2]: No need to do anything now with this.

File: static/js/map.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
+// const modisURL = "modis.csv";
+const modisURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_modis";
```
Comment (philipstubbs13) [inline static/js/map.js]: Make a note to remove these hardcoded urls (https://australia-fire-api-dashboard.herokuapp.com/api/v1.0) when we move the map to the flask app.

Fine for now. No need to do anything with this comment at the moment.

File: static/js/map.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+const viirsURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_viirs";
+const states = "https://raw.githubusercontent.com/rowanhogan/australian-states/master/states.geojson";
+const areaURL = "http://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_bystate";
+
+// TODO load in second dataset, create function that allows them to choose which dataset to use
+
+//////////// IMPORT THE DATA //////////////////
+function getData(modisURL, viirsURL) {
```
Comment (philipstubbs13) [inline static/js/map.js:10]: This is fine calling d3.json multiple times. JavaScript does have a Promise.all() method that makes making multiple api calls easier/cleaner. See the section on loading multiple files: http://learnjsdata.com/read_data.html

No need to make any changes now... just something to take note...

File: static/js/map.js [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+getData(modisURL, viirsURL);
+
+//////////// CALCULATE ANIMAL DEATHS PER STATE //////////////////
+// Uses general formula described in https://www.bbc.com/news/50986293
+
+// variable to hold calculations, to make later popover
+const deathsByState = [];
+
+function animalDeaths(hectareData, stateName) {
```
Comment (philipstubbs13) [inline static/js/map.js:48]: This is awesome! I'm slightly worried about performance doing this calculation on the frontend in the browser vs doing this on the backend in an endpoint. But, we can leave it and test it against the whole dataset later.

No need to make any changes now... just something to take note...

**Review by philipstubbs13 — APPROVED** [review]
Looks good.  I left a comment for future consideration, but no need to take action on those now. you can leave that one as is.

The only comment that should be addressed is to do a Kernal > Restart & Clear Output in the WebScraping.ipynb jupyter file. Once the output is cleared, feel free to merge.

File: static/js/map.js (hunk 2 above)
Comment (skonkiel) [reply static/js/map.js]: @philipstubbs13 thanks for catching the "clear output" thing on the Jupyter notebook. those are a pain!

I'm now at a point where the map is functional when running on the Flask server. To your point above: is the best way to add the endpoing URLs for the map.js file to have them use relative locations for the endpoints? I'm not quite sure how to approach that.

Comment (philipstubbs13) [reply static/js/map.js]: we can fix the endpoint urls tonight. I would just go ahead and merge what you have as long as it's working.

**Pull request merged**